.actrc

```diff
@@ -0,0 +1 @@
+-P ubuntu-20.04=catthehacker/ubuntu:full-20.04 --matrix os:ubuntu-20.04
```

Comment:
Please add empty line at the end of files like here

Reply:
I am probably removing this file -- seems act doesn't pick it up properly anyway. Will take your suggestion to move HOWTO as well, thanks @BartoszCiesla. Will update PR shortly.